EngineLayer/CrosslinkSearch/CrosslinkSpectralMatch.cs

```diff
@@ -18,15 +18,46 @@ public CrosslinkSpectralMatch(PeptideWithSetModifications theBestPeptide, int no
         public CrosslinkSpectralMatch BetaPeptide { get; set; }
```

Comment:
unit tests?

Reply:
This pull request is mainly on feature addition or bug fix. The unit tests are added based on existing unit tests.

Reply:
please try to get >90% unit test coverage